gitpod-io/openvsx
# Add scheduler to unblock jobs

## Description

- Query extension info inside ExtensionVersion job
- Add scheduler for Active and NamespaceVerified to remove chain jobs to unblock mirroring [internal chat](https://gitpod.slack.com/archives/C045J13P8A1/p1667237773173489?thread_ts=1667108727.309359&cid=C045J13P8A1)

## Related Issue(s)

Fixes #

## How to test

- Open this PR with Gitpod
- See `runServer` task loggings
- Exec `psql -h 127.0.0.1 -p 5432 -U gitpod` to connect to database
- Exec `\x on;` to get prettier print ()
- Exec `select * from extension;` to check if `arm.remote-build` is active after jobs are done (after Publish 4 extensions and ExtensionActiveJob done)
